assets/js/googlesitekit/modules/datastore/modules.js

```diff
@@ -641,6 +669,14 @@ const baseReducer = createReducer( ( state, { type, payload } ) => {
 			break;
 		}
 
+		case RECEIVE_INLINE_MODULES_DATA: {
+			const { inlineModulesData } = payload;
+			return {
+				...state,
+				inlineModulesData,
+			};
+		}
```

Comment:
Just to chip in here that we recently rewrote the reducer to use our immer wrapper `createReducer` which is creating a draft copy of the state, and we can mutate it directly (there is no need for a return value):
```suggestion
		case RECEIVE_INLINE_MODULES_DATA: {
			state.inlineModulesData = payload.inlineModulesData;
			break;
		}
```